### PR TITLE
Expand intro to generator in Migrations Guide

### PR DESCRIPTION
**Pull request by utilum**
[ci skip]

### CODE REVIEW

File: guides/source/active_record_migrations.md [hunk 1]
```diff
@@ -135,6 +135,16 @@ class AddPartNumberToProducts < ActiveRecord::Migration[5.0]
...
 
+This generator can do much more than make timestamps. Based on naming
+conventions and additional (optional) arguments it can also start fleshing
+out the migration.
+
+The migration name, the first argument passed to the generator, is
+composed of an action, an object, and an optional preposition + indirect
+object. In the example above the action is `Add`, the object is
+`PartNumber`, `To` is the preposition, and `Products` is the indirect
+object.
```
[the remark this answers is not in the document]
Comment (utilum) [reply guides/source/active_record_migrations.md]: Thank you for the feedback @vipulnsward . I've tried to fix the second part. Contemplating the first.

**Comment by utilum** [comment]
OK, I accept. Dropped the grammar paragraph and replaced XXX/YYY with Column/Table.
Edited the paragraph you quoted.
What do you think?

**Comment by oniofchaos** [comment]
👍

**Review by gmcgibbon — APPROVED** [review]
I like this change 👍 any further concerns @vipulnsward?

**Pull request merged**